src/main/webapp/script.js

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-google.charts.load('current', {'packages':['corechart']});
-google.charts.setOnLoadCallback(getChart);
-
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
+const HIGHEST_SCORE = 1.0;
+const LOWEST_SCORE = -1.0;
+const INTERVAL = 0.2;
```

Comment:
Nit: these 3 variables are identical and repeated through both this and the statistics class. In the future, we may want to consolidate them so that they're accessed from the same place.

Reply:
Spoke offline about a new PR with a constants file